configure

```diff
@@ -583,6 +618,12 @@ while [[ ! -z $1 ]] ; do
 #      HDF5LIB="-I$VALUE/include $VALUE/lib/libhdf5_hl.a $VALUE/lib/libhdf5.a -lm"
 #      echo "Using HDF5LIB: $HDF5LIB"
 #      ;;
+    "--with-readline")
```

Comment:
> Surely it's one of the most widely deployed packages in any linux -- isn't it like a dependency of the shell?

@drroe will have to weigh in here.  I know that Amber was already bundling and shipping a copy of readline as part of the deprecated and now-removed sleap program (a tleap rewrite that *was* better in many ways, except it was buggy and nobody used it so it never got debugged).  It may have just been simple to copy that over and add it to cpptraj.

Also, just because numerous tools had readline support doesn't mean the dev headers and libraries were installed on each system...  It's such a lightweight dependency that we probably thought "what's the harm?".  But Dan would have the final say on why it was done.